components/exercises/chapter-5-oscar/Flattening/Flattening.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+import Flattening, { phrases } from './Flattening';
+
+describe('Flattening', () => {
+  it('should render the correct number of phrases', () => {
+    render(<Flattening />);
+    for (const phrase of phrases) {
+      const phraseElement = screen.getByText(phrase.phrase);
```

Comment:
`phrase.phrase` is a code smell, this indicates some variable is not properly named

Reply:
refactored to phraseObject thank you